tests/test_meta.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import unicode_literals, absolute_import, division
+
+import re
+
+import pytest
+
+from pep517 import meta
+
+
+pep517_needs_python_3 = pytest.mark.xfail(
+    'sys.version_info < (3,)',
+    reason="pep517 cannot be built on Python 2",
```

Comment:
I'm working towards resolving this, but only in odd scraps of time at the moment.